### PR TITLE
ClimateMapped Payload Setup

### LINKED ISSUE #912: ClimateMapped PayloadCMS Setup
Setup PayloadCMS for ClimateMapped

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR setup Payload CMS for ClimateMappedAfrica

Fixes #912

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/climatemappedafrica/payload.config.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import seo from "@payloadcms/plugin-seo";
+import { sentry } from "@payloadcms/plugin-sentry";
+import { defaultLocale, locales } from "./src/payload/utils/locales";
+
+import Media from "./src/payload/collections/Media";
+import Pages from "./src/payload/collections/Pages";
+import Users from "./src/payload/collections/Users";
+
+dotenv.config();
+dotenv.config({ path: path.resolve(__dirname, "./.env.local") });
```
Comment (kilemensi) [inline apps/climatemappedafrica/payload.config.ts:21]: I _think_ we no longer need to use `dotenv`... `next/env` should be able to do everything we need (check **RoboShield**)

**Review by kilemensi — COMMENTED** [review]
Code LGTM!

**Review by koechkevin — APPROVED** [review]

**Review by m453h — APPROVED** [review]
Looks Good 🚀 

Not a show stopper but we could also use Aliases to make life easier in the future 💭

**Comment by kelvinkipruto** [comment]
> Not a show stopper but we could also use Aliases to make life easier in the future

I think I should try this before merging

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**